its/plugin/projects/yaml-aws-lambda/file.yaml

```
@@ -0,0 +1,16 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
+  SomeIamPolicy:
+    Type: AWS::IAM::Policy
+    Properties: 
+      PolicyDocument: 
+        Statement:
+          - Effect: Allow
+            Action: "*" # Noncompliant
```

If you want to test sonar-iac. I would recommend scanning for rule cloudformation:S6258
```suggestion
    Type: 'AWS::S3::Bucket' # Noncompliant
```
It's much simpler and does not need a bucket next to the policy